torch/utils/data/dataloader.py

```diff
@@ -69,7 +72,7 @@ def _worker_manager_loop(in_queue, out_queue, done_event, pin_memory):
             continue
         idx, batch = r
         try:
-            if pin_memory:
+            if pin_memory and torch.cuda.is_available():
```

Comment:
This seems unnecessary once you’ve done this in the constructor of the data loader

Reply:
Yes, thanks. Meant to get rid of that as I moved it to the constructor.